caput/time.py

```diff
@@ -408,6 +412,220 @@ def transit_RA(self, time):
 
         return ra
 
+    def transit_times(
+        self, source, t0, t1=None, step=0.2, lower=False, return_dec=False
+    ):
+        """Find the transit times of the given source in an interval.
+
+        Parameters
+        ----------
+        source : skyfield source
+            The source we are calculating the transit of.
+        t0 : float unix time, or datetime
+            The start time to search for. Any type that be converted to a UNIX time
+            by caput.
+        t1 : float unix time, or datetime, optional
+            The end time of the search interval. If not set, this is 1 day after the
+            start time `t0`.
+        step : float
+            The initial search step in days. This is used to find the approximate
+            location between transits, and should be set to something less than the
+            spacing between transits.
+        lower : bool, optional
+            By default this only returns the upper (regular) transit. This will cause
+            lower transits to be returned instead.
+        return_dec : bool, optional
+            If set, also return the declination of the source at transit.
+
+        Returns
+        -------
+        times : np.ndarray
+            UNIX times of transits.
+        dec : np.ndarray
+            Only returned if `return_dec` is set. Declination of source at transit.
+        """
+
+        # The function to find routes for. For the upper transit we just search for
+        # HA=0, for the lower transit we need to rotate the 180 -> -180 transition to
```

Comment:
What's HA?

Reply:
Hour angle. That one should be very well understood by astronomers.